varnish2.rb

```diff
@@ -0,0 +1,58 @@
+require 'formula'
+
+class Varnish2 < Formula
+  url 'http://repo.varnish-cache.org/source/varnish-2.1.5.tar.gz'
+  head 'http://repo.varnish-cache.org/source/varnish-3.0.0-beta1.tar.gz'
```

Comment:
We should probably remove this, since we don't want folks to accidentally install v3.